src/main/java/uk/co/ramp/covid/simulation/output/CsvOutput.java

```diff
@@ -23,28 +53,41 @@ public static void writeDailyStats(Appendable out, int startIterID, List<List<Da
         CSVPrinter printer = new CSVPrinter(out, CSVFormat.DEFAULT.withHeader(headers));
         for (int i = 0; i < stats.size(); i++) {
             for (DailyStats s : stats.get(i)) {
-                printer.printRecord(s.csvRecords(startIterID + i));
+                printer.printRecord(s.csvRecords(startIteration + i));
             }
         }
         printer.close();
     }
     
-    public static void writeDailyStats(Path outF, int startIterID, List<List<DailyStats>> stats) {
+    private void writeDailyStats() {
         try {
-            FileWriter out = new FileWriter(outF.toFile());
-            writeDailyStats(out, startIterID, stats);
+            FileWriter out = new FileWriter(outputDir.resolve(STATS_FNAME).toFile());
+            writeDailyStats(out);
             out.close();
         } catch (IOException e) {
             LOGGER.error(e);
         }
     }
+    
+    public String dailyStatsAsCSVString() {
+        StringWriter sw = new StringWriter();
+        try {
+            writeDailyStats(sw);
+        } catch (IOException e) {
```

Comment:
This method is only used in tests.  I think it should throw the IOException (and not catch it or log a warning).

Reply:
Sure! I also noticed it was only really for testing and didn't quite know what the best course of error handling was (mainly in case someone used it outside a test); but now you mention it I think just throwing it is fine.